.pipelines/azdo-base-pipeline.yml

```diff
@@ -10,6 +10,7 @@ steps:
   enabled: 'true'
   
 - script: |
+   pip install --user -r $(Build.SourcesDirectory)/tests/requirements.txt
```

Comment:
Let's put these into the main requirements.txt file

Reply:
Done